cli/makefile

```diff
@@ -16,20 +16,20 @@ all: linux darwin windows
 
```

Comment:
The fully qualified package names weren't required. We also removed a lot unused variables.